src/types/UiaTextRangeBase.hpp

```diff
@@ -218,7 +162,6 @@ namespace Microsoft::Console::Types
     protected:
         UiaTextRangeBase() = default;
 #if _DEBUG
-        void _outputRowConversions(IUiaData* pData);
         void _outputObjectState();
 #endif
         IUiaData* _pData;

```

Comment:
I can't set the comment on line 171 because it's outside the fold, but it turns out `_wordDelimiters` is a string. So assigning it the pointer (per my other comment from inside `RuntimeClassInitialize` is fine, but weird. You could just assign the whole `std::wstring_view` to the `std::wstring` instead of fetching the `.c_str()` out of it.

Reply:
(and probably note that you're implicitly taking a copy so the lifetime of the rvalue doesn't matter.)